packages/spaces/tests/Spaces.test.js

```diff
@@ -421,15 +396,14 @@ describe('Spaces', () => {
       expect(specificSpacesByConfigurationIds.textContent).toBe('Id: 2 Id: 3 ');
     });
 
-    it('returns spaces by payerId', async () => {
+    it('returns all matching spaces when searching by payerId', async () => {
```

Comment:
Need to this bugfix out as technically a breaking change. The order of `payerIDs` returned from the API is not guaranteed to always be the same, so returning 1 config based on the lowest index of matching `payerID` can't really be relied on. `useSpaces` will now return all associated configs that match a `payerID`

Reply:
done